src/MenuItem.php

```diff
@@ -136,7 +136,9 @@ protected function __construct(WP_Post $data, $menu = null)
         $this->wp_object = $data;
         $this->menu = $menu;
 
-        $data = (object) $data;
+        /**
```

Comment:
Why is this doc added over here instead of in the top of the class defining the property?

Reply:
I think we should probably add types to declared properties, especially for the very predictable ones.

Reply:
This is here because PHPStan couldn’t find a title property on the `WP_Post` object. The `title` property is set dynamically for menu items. I didn’t find an easier way to solve this :).